lib/hanami.rb

```diff
@@ -118,6 +118,30 @@ def self.app
     App.new(configuration, environment)
   end
 
+  # Check to see if application allowed to load.
+  #
+  # You can set allowed application in `HANAMI_APPS` env variable.
+  # If `HANAMI_APPS` variable is emply - any apps allowed.
+  #
+  # @return [TrueClass,FalseClass] the result of the check
+  #
+  # @since x.x.x
+  #
+  # @example
+  #
+  #   # Mount hanami app for specific app
+  #   Hanami.configure do
+  #     if Hanami.app?(:web)
+  #       require_relative '../apps/web/application'
+  #       mount Web::Application, at: '/'
+  #     end
```

Comment:
I suspect this chunk of code will be repeated, and might benefit from being extracted into some macro `load_app(name, path)` (naming could be improved though).